tests/db_migrator_input/config_db/cross_branch_upgrade_to_4_0_3_expected.json

```diff
@@ -3,17 +3,17 @@
         "VERSION": "version_4_0_3"
     },
     "FLEX_COUNTER_TABLE|ACL": {
-        "FLEX_COUNTER_STATUS": "true",
+        "FLEX_COUNTER_STATUS": "enable",
```

Comment:
>enable [](http://example.com/codeflow?start=32&length=6)

Just wondering, should we fix yang model, or fix the test data?

Reply:
I will check with owner.

Reply:
@vadymhlushko-mlnx has confirmed this is a mistake.